.drone.yml

```diff
@@ -10,22 +10,52 @@ pipeline:
     repo: sharpnotionsllc/pantry-list-frontend
     dockerfile: Dockerfile.build
     tags:
-      - latest
+      - 1.0.${DRONE_BUILD_NUMBER}
     secrets: [ docker_username, docker_password ]
     when:
       branch: master
       event: push
 
   deploy_now:
-    image: sharpnotionsllc/drone-now:latest
+    image: sharpnotionsllc/drone-now:1.0.${DRONE_BUILD_NUMBER}
     team: sharpnotions
     type: docker
     deploy_name: pantry-list-frontend
     alias: pantry-list-frontend
+    now_token: ${NOW_TOKEN}
     secrets: [ now_token ]
     cleanup: true
     public: true
     pull: true
     when:
       branch: master
       event: push
+
+  build-push-prod:
+    image: plugins/docker
+    repo: sharpnotionsllc/pantry-list-frontend
```

Comment:
Isn't this the image we want to use `DRONE_BUILD_NUMBER` on? I don't think we need to build and push a new drone-now deploy image for ever merge to master

Reply:
Thats the best practice is to deploy a new Tagged image every master merge.